bot/validations.ts

```diff
@@ -306,20 +326,20 @@ const validateInvoice = async (ctx, lnInvoice) => {
   }
 };
 
-const isValidInvoice = async (ctx, lnInvoice) => {
+const isValidInvoice = async (ctx: MainContext, lnInvoice: string) => {
   try {
     const invoice = parsePaymentRequest({ request: lnInvoice });
     const latestDate = new Date(
-      Date.now() + parseInt(process.env.INVOICE_EXPIRATION_WINDOW)
+      Date.now() + Number(process.env.INVOICE_EXPIRATION_WINDOW)
     ).toISOString();
-    if (!!invoice.tokens && invoice.tokens < process.env.MIN_PAYMENT_AMT) {
+    if (!!invoice.tokens && invoice.tokens < Number(process.env.MIN_PAYMENT_AMT)) {
```

Comment:
@Mersho this looks like a bug that is existing upstream? we should call parseFloat() to fix the bug first AFAIU